frontend/src/components/pages/Scheduling/SelectDateTime.tsx

```diff
@@ -9,12 +10,31 @@ const SelectDateTime = ({
   navigation,
 }: SchedulingStepProps) => {
   const { previous, next } = navigation;
-  //   const { insert form fields for this page here } = formData;
+  const { daypart } = formValues;
+
+  const dayparts = [
+    "Early Morning (12am - 6am)",
+    "Morning (6am - 11am)",
+    "Afternoon (11pm - 4pm)",
+    "Evening (4pm - 9pm)",
+    "Night (9pm - 12am)",
+  ];
+
+  const handleChange = (e: any, name: string) => {
+    setForm({ target: { name, value: e } });
+  };
 
   return (
-    // Insert Select Date and Time page here
     <Container>
       <Text textStyle="mobileHeader2"> This is Date and Time page </Text>
+      <RadioSelectGroup
+        name="daypart"
+        label="What time of day would you like to drop off your donation?"
+        value={daypart}
+        values={dayparts}
+        isRequired
+        onChange={(e: any) => handleChange(e, "daypart")}
```

Comment:
instead of any, I believe it should be `e: React.ChangeEvent<HTMLInputElement>`, same in `handleChange` function.